frontstage/common/request_handler.py

```diff
@@ -0,0 +1,32 @@
+import logging
```

Comment:
Why do we need a request handler?  If it's not too difficult, calls should be in the controller rather than passing it off to a helper module. 
If it's too much hassle for this PR (which is pretty big to be fair), could you make a card for it?

Reply:
@insacuri I would much prefer to use the requests lib (which is already an abstraction around urllib) directly but this was dropped in to speed up the unpicking of the ported code. 

I'll see what can be done. Consistency curse strikes again!

Reply:
@insacuri Updated to remove the wrapper in latest commit.